src/components/pages/article/tabs/fulltext-tab.tsx

```diff
@@ -28,7 +28,7 @@ export const ArticleFullTextTab = (props: { metaData: MetaData, content: Content
         { props.peerReview && <ReviewContent content={props.peerReview.evaluationSummary.text} isAssessment={true} peerReviewUrl={props.peerReviewUrl}/> }
         <ArticleContent content={props.content} />
         <ReferenceList references={props.metaData.references} />
-        <AuthorInformationList authors={props.metaData.authors}/>
+        <AuthorInformationList authors={props.metaData.authors} license={props.metaData.license} publishedYear={props.metaData.publishedYear} />
```

Comment:
The publishedYear is probably not the correct year to display for copyright reasons. A version 2 (or version 23) would potential be one of the years previous to when the preprint was published. 

My guess, you'd need to dig into the `ArticleStatus.timeline` but that's just off the top of my head. It's not going to be a pretty solution. Happy to chat it through.